racetrack_commons/racetrack_commons/database/database.py

```diff
@@ -0,0 +1,41 @@
+import os
+from pathlib import Path
+from typing import Dict
+
+
+def populate_database_settings(base_dir: Path) -> Dict[str, Dict]:
```

Comment:
Since it's quite long function, how about adding unit tests for the logic of constructing database_name? That could be crudely done manipulating os.env['POSTGRES_DB'] etc, or extracting that part as function.

Reply:
Right. I've added unit test